src/runtime.cpp

```diff
@@ -92,7 +92,7 @@ namespace hpx {
         }
     }
 
-    HPX_EXPORT BOOL WINAPI termination_handler(DWORD ctrl_type)
+    BOOL WINAPI termination_handler(DWORD ctrl_type)
```

Comment:
The symbols in this file that are now unmarked should be exported. We likely should mark them at their declaration.

Reply:
Good point. This one is actually only `extern` declared in `hpx_init.cpp`, so it could probably do with a proper declaration in the header file.

Reply:
Most of the other functions are already marked to be exported in header files. I need to verify that they actually get included here though.